cmd/security-file-token-provider/res/token-config.json

```diff
@@ -8,6 +8,12 @@
             "list",
             "read"
           ]
+        },
+        "secret/edgex/core-command/mqtt-bus": {
+          "capabilities": [
+            "list",
+            "read"
+          ]
```

Comment:
It's time to get rid of the custom policy here.

The default policy (use_edgex_defaults) already grants read/write to all of secret/edgex/core-command/*

We only need to have a custom policy for any secrets that aren't under <service-key>